chart/templates/storage-admin/deployment.yaml

```diff
@@ -17,7 +17,17 @@ spec:
     metadata:
       labels: {{ include "labels.storageAdmin" . | nindent 8 }}
     spec:
+      initContainers:
+        {{ include "initContainerAssets" . | nindent 8 }}
+        {{ include "initContainerCache" . | nindent 8 }}
+        {{ include "initContainerParquetMetadata" . | nindent 8 }}
+        {{ include "initContainerDuckDBIndex" . | nindent 8 }}
+        {{ include "initContainerDescriptiveStatistics" . | nindent 8 }}
       containers: {{ include "containerStorageAdmin" . | nindent 8 }}
       nodeSelector: {{ toYaml .Values.storageAdmin.nodeSelector | nindent 8 }}
       tolerations: {{ toYaml .Values.storageAdmin.tolerations | nindent 8 }}
-      volumes: {{ include "volumeData" . | nindent 6 }}
+      volumes:
+        {{ include "volumeCache" . | nindent 8 }}
+        {{ include "volumeData" . | nindent 8 }}
+        {{ include "volumeDuckDB" . | nindent 8 }}
```

Comment:
Maybe we need `{{ include "volumeMountParquetMetadataRW" . | nindent 8 }}` also?

Reply:
no!!!

Reply:
In fact, it's included in `{{ include "volumeData" . | nindent 8 }}` because we didn't move the data to its own EFS, so it's the same volume as assets